backends/dnf/meson.build

```diff
@@ -20,6 +22,26 @@ install_data(
   install_dir: join_paths(python_package_dir),
 )
 
+add_languages('cpp')
+shared_module(
+  'notify_packagekit',
+  'notify_packagekit.cpp',
+  cpp_args: '-std=c++20',
+  include_directories: packagekit_glib2_includes,
+  dependencies: [
+    dnf5_dep,
```

Comment:
anyone familiar with meson can tell me why i don't actually need to add the dnf5 dependency to the module? it seems like headers are propagated globally? that's weird...

Reply:
Ah well, because they're installed in a default includes directory